src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonSerializerSettingsProvider.cs

```diff
@@ -20,6 +21,11 @@ public static JsonSerializerSettings CreateSerializerSettings()
         {
             return new JsonSerializerSettings
             {
+                ContractResolver = new DefaultContractResolver
```

Comment:
We don't test for default values and we rarely write unit tests for simple `static` classes. In this case, 26 functional tests failed when I changed the default naming strategy. That'll happen again if the defaults change.